bok_choy/page_object.py

```diff
@@ -534,6 +534,18 @@ def axs_audit_rules_to_run(self):
         """
         return []
 
+    def axs_audit_rules_to_ignore(self):
+        """
+        List of rules to ignore for accessibility errors on the page.
+        See https://github.com/GoogleChrome/accessibility-developer-tools/tree/master/src/audits
+
+        E.g. return ['badAriaAttributeValue']
+        An empty list means to run rules as defined by axs_audit_rules_to_run.
+        Otherwise, if rules are listed here, they will be ignored even if
+        they are specified in axs_audit_rules_to_run.
+        """
+        return None
```

Comment:
Yeah, that seems best to me. Set it to [] here and take None out of here and accessibility.rst (and the tests, if it is in any).